src/styles/blocks/daysNumbering.scss

```diff
@@ -0,0 +1,11 @@
+.calendar {
+  @for $number from 1 through 31 {
```

Comment:
dont use it in separate files write it only in calendar.scss. It's not necessary to separate logic which nested in one class